addon/adapters/batch-request.js

```diff
@@ -22,31 +27,31 @@ export default JSONAPIAdapter.extend({
   },
 
   batchCreate(items, options) {
-    if (isPresent(options) && isPresent(options.skipStoreUpdate)) {
-      return this._batch(items, 'POST', options.skipStoreUpdate);
-    }
+    options = options || {};
+    let httpMethod = options.httpMethod || 'POST';
+    let skipStoreUpdate = options.skipStoreUpdate || false;
 
-    return this._batch(items, 'POST', false);
+    return this._batch(items, ACTION_NAME.CRAETE, httpMethod, skipStoreUpdate, options.getIdsFunc);
   },
 
   batchUpdate(items, options) {
-    if (isPresent(options) && isPresent(options.skipStoreUpdate)) {
-      return this._batch(items, 'PATCH', options.skipStoreUpdate);
-    }
+    options = options || {};
+    let httpMethod = options.httpMethod || 'PATCH';
+    let skipStoreUpdate = options.skipStoreUpdate || false;
 
-    return this._batch(items, 'PATCH', false);
+    return this._batch(items, ACTION_NAME.UPDATE, httpMethod, skipStoreUpdate, options.getIdsFunc);
```

Comment:
I think `options.getIdsFunc` is probably a good feature but should be in a separate PR